code/modules/events/holiday/xmas.dm

```diff
@@ -11,8 +11,10 @@
 	for(var/obj/structure/flora/tree/pine/xmas/xmas in world)
 		if(!(xmas.z in GLOB.station_z_levels))
 			continue
-		xmas.icon_state = "pinepresents"
-		xmas.gifts_under_tree = TRUE
+
+		new /obj/structure/flora/tree/pine/xmas/presents(get_turf(xmas))
```

Comment:
xmas.loc

Reply:
Why?

Reply:
When you transform something you want it to appear exactly where the old one was

Reply:
good point